BasicsCodelab/app/src/main/java/com/codelab/basics/MainActivity.kt

```diff
@@ -76,9 +77,28 @@ fun MyScreenContent(names: List<String> = listOf("Android", "there")) {
     }
 }
 
+@Composable
+fun NameList(modifier: Modifier, names: List<String>) {
+    LazyColumn(modifier = modifier) {
+        items(items = names) { name ->
+            Greeting(name = name)
+            Divider(color = Color.Black)
+        }
+    }
+}
+
 @Composable
 fun Greeting(name: String) {
-    Text(text = "Hello $name!", modifier = Modifier.padding(24.dp))
+    var isPressed by remember { mutableStateOf(false) }
```

Comment:
What about `isSelected`? `isPressed` feels like you're expecting people to long press in it for it to be pressed
```suggestion
    var isSelected by remember { mutableStateOf(false) }
```